sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md

```diff
@@ -111,4 +144,133 @@ directive:
         delete $["MetadataFilter"]["properties"]["metadata"]["items"]["$ref"];
         $["MetadataFilter"]["properties"]["metadata"]["items"]["type"] = "object";
         delete $["MetadataRecord"];
-```
\ No newline at end of file
+```
+
+### Authoring
+
+
+#### Remove operation group
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $["paths"]["/query-knowledgebases/projects"]["get"]
+    transform: >
+        $["operationId"] = "listProjects";
+  - from: swagger-document
+    where: $["paths"]["/query-knowledgebases/projects/{projectName}"]["get"]
+    transform: >
+        $["operationId"] = "getProjectDetails";
+  - from: swagger-document
+    where: $["paths"]["/query-knowledgebases/projects/{projectName}"]["patch"]
+    transform: >
+        $["operationId"] = "createProject";
+  - from: swagger-document
+    where: $["paths"]["/query-knowledgebases/projects/{projectName}"]["delete"]
+    transform: >
+        $["operationId"] = "deleteProject";
+  - from: swagger-document
+    where: $["paths"]["/query-knowledgebases/projects/{projectName}/:export"]["post"]
```

Comment:
you've already done the directive, but in the future you can use built in directives to save yourself some code and to make the readme easier to read https://github.com/Azure/autorest/blob/main/docs/generate/built-in-directives.md#rename-operation